src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+
+    const lowerCaseTitle = title.toLowerCase();
+    const lowerCaseDescription = description.toLowerCase();
+    const lowerCaseQuery = query.toLowerCase();
```

Comment:
Good job, that is really nice thing to create variables to improve readability
You could do it simpler though. 

lets make but 2 conditions though. 
If you do it like `const doesTitleMatch = title.toLowerCase().includes(query.toLowerCase())` it will be even more readable and simple. Return will look more clear too.